src/WebAppDIRAC/Lib/WebHandler.py

```diff
@@ -345,14 +293,11 @@ def _authzSESSION(self):
     def getLog(cls):
         return sLog
 
-    def getCurrentSession(self):
-        return self.__session
-
     def getUserSetup(self):
         return self.__setup
 
     def getSessionData(self):
-        if not self.__sessionData:
+        if not hasattr(self, "__sessionData"):
```

Comment:
is this one of those points where `ObjectLoader` should be used?

Reply:
I don't think so?